src/main/scala/nl.knaw.dans.easy.validatebag/rules/metadata/package.scala

```diff
@@ -20,24 +20,28 @@ import java.nio.ByteBuffer
 import java.nio.charset.{ CharacterCodingException, Charset }
 import java.nio.file.{ Path, Paths }
 
-import nl.knaw.dans.easy.validatebag.{ TargetBag, XmlValidator }
 import nl.knaw.dans.easy.validatebag.validation._
+import nl.knaw.dans.easy.validatebag.{ TargetBag, XmlValidator }
 import nl.knaw.dans.lib.error._
 import nl.knaw.dans.lib.logging.DebugEnhancedLogging
 
+import scala.util.matching.Regex
 import scala.util.{ Success, Try }
 import scala.xml._
 
 package object metadata extends DebugEnhancedLogging {
   val filesXmlNamespace = "http://easy.dans.knaw.nl/schemas/bag/metadata/files/"
   val dcxDaiNamespace = "http://easy.dans.knaw.nl/schemas/dcx/dai/"
+  val identifierTypeNamespace = "http://easy.dans.knaw.nl/schemas/vocab/identifier-type/"
   val gmlNamespace = "http://www.opengis.net/gml"
   val dcNamespace = "http://purl.org/dc/elements/1.1/"
   val dctermsNamespace = "http://purl.org/dc/terms/"
   val schemaInstanceNamespace = "http://www.w3.org/2001/XMLSchema-instance"
 
   val allowedFilesXmlNamespaces = List(dcNamespace, dctermsNamespace)
 
+  val doiPattern: Regex = raw"^(https://doi.org/)?10(\.\d+)+/.+".r
```

Comment:
- I wrote the regex myself, based on the information I could find in https://www.doi.org
- About the supported characters, the site says: `DOI names may incorporate any printable characters from the Universal Character Set (UCS-2), of ISO/IEC 10646, which is the character set defined by Unicode.`
- I had understood from Linda that DOIs in URLs would also be possible, but it may be that I have misunderstood her.  Perhaps she just meant to say that DOI in URL is possible, not necessarily that we are supporting that feature.  Anyway, I have now changed the code so that DOIs in URLs are not accepted.  I also added a check to test this in a Unit test.